docs/src/pages/guides/right-to-left/right-to-left.md

```diff
@@ -33,10 +33,10 @@ npm install jss-rtl
 If you are using `emotion` or `styled-components`, you need this stylis plugin to flip the styles: [stylis-plugin-rtl](https://github.com/styled-components/stylis-plugin-rtl).
 
 ```sh
-npm install stylis-plugin-rtl@^1
+npm install stylis-plugin-rtl
 ```
 
-Note: both `emotion` and `styled-components` currently work with the v1 of the plugin.
+**Note**: Currently `emotion` is compatible with v2 of the plugin. However, `styled-components` currently work with the v1 of the plugin, so if you are using `styled-components` as styled engine, make sure to install the correct version.
```

Comment:
```suggestion
**Note**: Only `emotion` is compatible with version 2 of the plugin. `styled-components` requires version 1. If you are using `styled-components` as styled engine, make sure to install the correct version.
```

- most important fact at the start (only emotion supports the latest)
- shorter sentences
- reduce jargon